manta-pay/src/test/payment.rs

```diff
@@ -55,16 +55,13 @@ where
         .expect("Unable to build MINT proof.")
 }
 
-/// Samples context of [`Mint`]s for individual assets.
-/// The context is used in `prove_private_transfer` function and
-///     `prove_reclaim` for benchmarking.
-/// Note that the mint proof is not included in the context, since it
-///     is not used when proving private transfer or reclaim.
-/// The generation of mint proof is also not included so that this function
-///     is lightweight and we can precisely profile the performance of
-///     `prove_priate_transfer` and `prove_reclaim`.
+/// Samples a [`Mint`] spender.
```

Comment:
The reason it shouldn't be `context` is because that has a specific meaning `ProvingContext` and `VerifyingContext`. Also, `spending_key` and `sender` are the two required pieces to _spend_ the asset, so `spender` matches the semantics here.